### ISSUE TITLE
number of unread messages in channel list not being read by screen reader

### ISSUE THREAD

**Issue by h-rezaei** [thelounge/thelounge#2829]
My screen reader is not able to read the number of unread messages that appears in the channel list.
So it reads the channels  fine, but no numbers after them.
I'm on thelounge v3 prerelease

**Comment by xPaw** [comment]
This should be fixed by #2828

**Comment by h-rezaei** [comment]
I have added that fix and still my screen reader (NVDA)  won't read any unread counts.
just to be sure, when i used the github checkout command i got this:
Note: checking out 'origin/xpaw/chanlist-unread-label'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at d087c726 Put unread and highlight counts in aria-label in the channel list

then i typed yarn install, which at the end gave me a bunch of errors about sqlite3, but after that thelounge ran successfully. It still tells me that sqlite module is missing though.

**Comment by dgw** [comment]
#2828 changes a file in `client`, so you'll need to run `npm run build` first to compile the changes, IIRC.

**Comment by h-rezaei** [comment]
ignore my previous comment. this works perfectly with NVDA on chrome and firefox.
But what made me confuse, is that the unread number count only appears or updates whenever i refresh my browser. I don't know if this behavior is intended

**Comment by astorije** [comment]
That's probably a bug, might be a different one. @xPaw, mind taking a look?

**Comment by xPaw** [comment]
Yeah it's a but because I forgot to write jQuery code to update it, since handlebars is not reactive :(
